src/runner.rs

```diff
@@ -57,6 +57,8 @@ pub fn run(
     let mut toastfile_dir = PathBuf::from(&settings.toastfile_path);
     toastfile_dir.pop();
 
+    let output_dir = PathBuf::from(&settings.output_path);
```

Comment:
Do we need this owned copy of the path? It seems like we can just pass in `settings.output_path` directly where it's used below, since it's passed by reference to `docker::copy_from_container`.